src/App.js

```diff
@@ -1,13 +1,25 @@
 import React from "react";
 
 import Bookings from "./Bookings";
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristCards";
+import Footer from "./Footer";
 import "./App.css";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      {/* <header className="App-header">CYF Hotel</header> */}
```

Comment:
It looks like you've left quite a bit of commented out code around - I'd suggest you delete it - if you really need to get it back, your git history still has a copy, but for now it's mostly just adding noise to your code :)